<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpTestDialog</class>
 <widget class="QDialog" name="UdpTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDP测试工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="sourceAdapterLabel">
       <property name="text">
        <string>源适配器:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="sourceAdapterComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="sourceIpLabel">
       <property name="text">
        <string>源IP地址:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="sourceIpEdit">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
       <property name="placeholderText">
        <string>例如: 192.168.1.1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="sourcePortLabel">
       <property name="text">
        <string>源端口:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="sourcePortSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>12345</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="destAdapterLabel">
       <property name="text">
        <string>目标适配器:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="destAdapterComboBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="destIpLabel">
       <property name="text">
        <string>目标IP地址:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="destIpEdit">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
       <property name="placeholderText">
        <string>例如: 192.168.1.2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="destPortLabel">
       <property name="text">
        <string>目标端口:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="destPortSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>8080</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="payloadLabel">
     <property name="text">
      <string>发送内容:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="payloadEdit">
     <property name="placeholderText">
      <string>输入要发送的UDP数据包内容</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="hexModeCheckBox">
     <property name="text">
      <string>十六进制模式 (例如: 48 65 6C 6C 6F)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>发送数据包</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>准备就绪</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>UdpTestDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>